database/postgres/postgres_test.go

```diff
@@ -155,6 +155,39 @@ func TestMultiStatement(t *testing.T) {
 	})
 }
 
+func TestMultipleStatementsInMultiStatementMode(t *testing.T) {
+	dktesting.ParallelTest(t, specs, func(t *testing.T, c dktest.ContainerInfo) {
+		ip, port, err := c.FirstPort()
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		addr := pgConnectionString(ip, port) + "&x-multi-statement=true"
```

Comment:
Instead of modifying the returned connection string, do all of the url/addr building in `pgConnectionString()`